### ISSUE TITLE
member-ordering documentation does not explain what an instance sandwich is

### ISSUE THREAD

**Issue by grantwwu** [palantir/tslint#4051]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__: CLI 

#### TypeScript code being linted

N/A

with `tslint.json` configuration:

N/A

#### Actual behavior

https://palantir.github.io/tslint/rules/member-ordering/ does not explain what "instance-sandwich" does

#### Expected behavior

https://palantir.github.io/tslint/rules/member-ordering/ explains what "instance-sandwich" does

**Comment by kylecordes** [comment]
I found this quite strange also, it must've been just an oversight by whoever coded it. The meaning can be found by looking in the source code in the meantime:

https://github.com/palantir/tslint/blob/master/src/rules/memberOrderingRule.ts#L62

**Comment by FiretronP75** [comment]
**I also want to know what an instance-sandwich is!**

And I think fields-first and statics-first should also be defined in the docs. It isn't clear that they don't bring in any other rules with them besides the obvious. Does fields-first only check for fields coming first, or might it also put public before private? It should explicitly state that it does nothing more.

**Comment by grantwwu** [comment]
I made a PR for this.  How do I get someone to look at it?

**Comment by sibelius** [comment]
member-ordering broke some code for me

I had `state` in my class below  `getValue` method

```jsx
state = {
    input: '',
    search: this.getValue() ? this.getValue().name : '',
    isFetching: false,
  };
```

so `member-ordered` moved `state` to the top and broke my code

**Comment by grantwwu** [comment]
Please make a new issue; that appears entirely unrelated to this thread.
